content/manuals/security/for-admins/hardened-desktop/settings-management/compliance-reporting.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Understand how to use the Desktop settings reporting dashboard
+keywords: Settings Management, docker desktop, hardened desktop, reporting, compliance
+title: Desktop settings reporting
+linkTitle: Desktop Settings Reporting
+weight: 30
+---
+
+Desktop settings reporting is a feature of Desktop Settings Management that tracks and reports user compliance with the settings that are assigned to them. This lets administrators to track the application of settings and monitor what actions they need to take to make users compliant.
+
+This guide provides steps for accessing the reporting dashboard, viewing compliance status, and resolving non-compliant users.
+
+## Access Desktop settings reporting
+
+1. Sign in to the [Admin Console](https://app.docker.com/admin).
+2. Select your **organization** from the **Choose profile** page.
+3. Under Docker Desktop, select **Reporting**.
+
+This opens the Desktop settings reporting dashboard. From here you can:
+
+- Use the **Search bar** to search by username or email address
+- Filter by policies
+- Hide or un-hide compliant users
+- View a user’s compliance status and what policy is assigned to the user
+- Download a CSV file of user compliance information
+
+## View compliance status
+
+1. Sign in to the [Admin Console](https://app.docker.com/admin).
+2. Select your **organization** from the **Choose profile** page.
+3. Under Docker Desktop, select **Reporting**.
```

Comment:
**Question:** Do we need to repeat the same steps from the previous section?

Reply:
I typically try to include all steps for each section as if the user was looking at only that section. It helps Kapa if a user is using Docs AI search to find specific steps (ex: user searches how to view compliance status) and it helps if a user navigates directly to a heading instead of reading through the doc in its entirety